gradio/cli/commands/components/docs.py

```diff
@@ -77,69 +77,114 @@ def _docs(
+        )
+    module = importlib.import_module(name)
+    (docs, type_mode) = extract_docstrings(module)
 
-        if generate_space:
+    if generate_space:
+        if not simple:
             live.update(":computer: [blue]Generating space.[/]")
 
-            source = make_space(
-                docs=docs,
-                name=name,
-                description=description,
-                local_version=local_version
-                if local_version is None
-                else str(local_version),
-                demo=demo,
-                space=space if space is None else str(space),
-                repo=repo if repo is None else str(repo),
-                pypi_exists=pypi_exists,
-                suppress_demo_check=suppress_demo_check,
-            )
+        source = make_space(
+            docs=docs,
+            name=name,
+            description=description,
+            local_version=local_version
+            if local_version is None
+            else str(local_version),
+            demo=demo,
+            space=space if space is None else str(space),
+            repo=repo if repo is None else str(repo),
+            pypi_exists=pypi_exists,
+            suppress_demo_check=suppress_demo_check,
+        )
 
-            with open(_demo_dir / "space.py", "w") as f:
-                f.write(source)
+        with open(_demo_dir / "space.py", "w") as f:
+            f.write(source)
+            if not simple:
                 live.update(
                     f":white_check_mark: Space created in [orange3]{_demo_dir}/space.py[/]\n"
                 )
-            with open(_demo_dir / "css.css", "w") as f:
-                f.write(css)
+        with open(_demo_dir / "css.css", "w") as f:
+            f.write(css)
 
-        if generate_readme:
+    if generate_readme:
+        if not simple:
             live.update(":pencil: [blue]Generating README.[/]")
-            readme = make_markdown(
-                docs, name, description, local_version, demo, space, repo, pypi_exists
-            )
+        readme = make_markdown(
+            docs, name, description, local_version, demo, space, repo, pypi_exists
+        )
 
-            with open(_readme_path, "w") as f:
-                f.write(readme)
+        with open(_readme_path, "w") as f:
+            f.write(readme)
+            if not simple:
                 live.update(
                     f":white_check_mark: README generated in [orange3]{_readme_path}[/]"
                 )
+    if simple:
+        short_readme_path = Path(_readme_path).relative_to(_component_dir)
+        short_demo_path = Path(_demo_path).relative_to(_component_dir)
+        live.update(
+            f":white_check_mark: Documention generated in [orange3]{short_demo_path}[/] and [orange3]{short_readme_path}[/]. Pass --no-generate-docs to disable auto documentation."
```

Comment:
This should be `space.py` not `demo.py` no?

<img width="564" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/3e829d0b-577e-4654-ac98-88ac81d67293">

Reply:
fixed!